### ISSUE TITLE
`mono` entries affect color scheme

### ISSUE THREAD

**Issue by jpalus** [neomutt/neomutt#4079]
## Expected Behaviour

In color terminal `mono` entries should be effectively ignored.

## Actual Behaviour

`mono` entries influence color theme.

## Steps to Reproduce

Have a mailbox with a new mail and use following color config (based on solarized-dark-16):
```
color normal        brightyellow    default       
color indicator     brightblack     yellow           
mono  indicator     reverse 
color index         blue            default         "~N"   
```
Highlighted new message with `mono indicator reverse`:
![image](https://github.com/neomutt/neomutt/assets/484155/3a94f638-0f7f-4150-b4a2-a9aa88fa4154)

Highlighted new message without `mono indicator reverse`:
![image](https://github.com/neomutt/neomutt/assets/484155/bb4d5539-18a8-453b-847d-6b34f8fcec5e)

## How often does this happen?

- Always

## When did it start to happen?

After upgrade to `neomutt 20231023`. `neomutt 20231006` worked fine.

## NeoMutt Version

```
NeoMutt 20231023
Copyright (C) 1996-2022 Michael R. Elkins and others.
NeoMutt comes with ABSOLUTELY NO WARRANTY; for details type 'neomutt -vv'.
NeoMutt is free software, and you are welcome to redistribute it
under certain conditions; type 'neomutt -vv' for details.

System: Linux 6.5.7-1 (aarch64)
ncurses: ncurses 6.4.20231001 (compiled with 6.4.20231001)
libidn2: 2.3.4 (compiled with 2.3.4)
GPGME: 1.22.0
OpenSSL: OpenSSL 3.1.2 1 Aug 2023
PCRE2: 10.42 2022-12-11
storage: bdb, lmdb
compression: lz4, zlib, zstd

Configure options: {LDFLAGS=-Wl,--as-needed -Wl,--no-copy-dt-needed-entries -Wl,-z,relro -Wl,-z,combreloc -Wl,-fuse-ld=mold } {CFLAGS=-O2 -fwrapv -pipe -Wformat -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fstack-protector-strong --param=ssp-buffer-size=4 -Werror=trampolines -fomit-frame-pointer -mbranch-protection=standard -mtune=cortex-a72.cortex-a53 } {CXXFLAGS=-O2 -fwrapv -pipe -Wformat -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fstack-protector-strong --param=ssp-buffer-size=4 -Werror=trampolines -fomit-frame-pointer -mbranch-protection=standard -mtune=cortex-a72.cortex-a53 } {FFLAGS=-O2 -fwrapv -pipe -Wformat -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fstack-protector-strong --param=ssp-buffer-size=4 -Werror=trampolines -fomit-frame-pointer -mbranch-protection=standard -mtune=cortex-a72.cortex-a53 } {FCFLAGS=-O2 -fwrapv -pipe -Wformat -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fstack-protector-strong --param=ssp-buffer-size=4 -Werror=trampolines -fomit-frame-pointer -mbranch-protection=standard -mtune=cortex-a72.cortex-a53 } CPPFLAGS= CC=/usr/libexec/icecc/bin/gcc CXX=/usr/libexec/icecc/bin/g++ --host=aarch64-pld-linux --build=aarch64-pld-linux --prefix=/usr --disable-debug --gpgme --bdb --with-bdb=/usr --lmdb --lua --lz4 --with-mailpath=/var/mail --mixmaster --pcre2 --sasl --ssl --zlib --zstd

Compilation CFLAGS: -O2 -fwrapv -pipe -Wformat -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fstack-protector-strong --param=ssp-buffer-size=4 -Werror=trampolines -fomit-frame-pointer -mbranch-protection=standard -mtune=cortex-a72.cortex-a53  -fno-delete-null-pointer-checks -D_ALL_SOURCE=1 -D_GNU_SOURCE=1 -D__EXTENSIONS__ -D_XOPEN_SOURCE_EXTENDED -I/usr/include/lua5.4 -I/usr/include -DNCURSES_WIDECHAR -I/include -I/usr/include/ -I/usr/include -O2

Default options:
  +attach_headers_color +compose_to_sender +compress +cond_date +debug 
  +encrypt_to_self +forgotten_attachments +forwref +ifdef +imap +index_color 
  +initials +limit_current_thread +multiple_fcc +nested_if +new_mail +nntp +pop 
  +progress +quasi_delete +regcomp +reply_with_xorig +sensible_browser +sidebar 
  +skip_quoted +smtp +status_color +timeout +tls_sni +trash 

Compile options:
  -autocrypt +fcntl -flock -fmemopen +futimens +getaddrinfo -gnutls +gpgme 
  -gsasl -gss +hcache -homespool +idn +inotify -locales_hack +lua +mixmaster 
  +nls -notmuch +openssl +pcre2 +pgp +sasl +smime -sqlite +sun_attachment 
  +truecolor 

MAILPATH="/var/mail"
MIXMASTER="mixmaster"
PKGDATADIR="/usr/share/neomutt"
SENDMAIL="/usr/sbin/sendmail"
SYSCONFDIR="/etc"

To learn more about NeoMutt, visit: https://neomutt.org
If you find a bug in NeoMutt, please raise an issue at:
    https://github.com/neomutt/neomutt/issues
or send an email to: <neomutt-devel@neomutt.org>
```

**Comment by flatcap** [comment]
Sorry, that's irritating.
As you've already guessed, commenting out the `mono` lines is the temporary workaround.

The code change that caused the problem is part one of a set of colour improvements.
It was _meant_ to be safe :-)

I made a quick fix for the release, but I've nearly finished colour work,
so I'm probably going to merge that and make a new release next week.

**Comment by flatcap** [comment]
Colour fixed merged in f9c11d60d.
